components/Navbar.vue

```diff
@@ -18,7 +18,7 @@
           class="lg:!hidden flex flex-grow items-center justify-end"
           @click="closeBurgerMenu">
           <NeoButton
-            v-if="isMobileNavbarOpen || isTinyMobile"
+            v-if="isMobileNavbarOpen || isTouch"
```

Comment:
```suggestion
```

this is not needed parent class has already `lg:!hidden`

same form `<Search />` component near it